experimental/Matrix/matrix.jl

```diff
@@ -0,0 +1,10 @@
+module MatrixGroupExperimental
```

Comment:
Maybe rename the directory to `MatrixGroup` or so (`MatrixGroups`, plural? Doesn't matter, this is only temporarily in experimental anyway). Then the test file could get the matching name `MatrixGroup-test.jl` (the "Experimental" part is a bit redundant :-)).
And then this module could be
```suggestion
module MatrixGroup
```